### PR TITLE
feat: added styles for table on mobile

### PR DESCRIPTION
**Pull request by CzarekDryl**
https://tw.auroracreation.com/app/tasks/15583746

image

### CODE REVIEW

File: src/components/v5/common/Table/types.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
-import { TableOptions } from '@tanstack/react-table';
+import { TableOptions, Row } from '@tanstack/react-table';
 
 export interface TableProps<T>
   extends Omit<TableOptions<T>, 'getCoreRowModel'> {
   getCoreRowModel?: TableOptions<T>['getCoreRowModel'];
   className?: string;
+  getRowClassName?: (row: Row<T>) => string | undefined;
```
Comment (krawczykkatarzyna) [inline src/components/v5/common/Table/types.ts:7]: getRowClassName?: (row: Row<T>) => string;

File: src/components/v5/common/TableWithMeatballMenu/TableWithMeatballMenu.tsx [hunk 2]
```diff
@@ -17,7 +17,15 @@ const TableWithMeatballMenu = <T,>({
     [columns, helper, getMenuProps],
   );
 
-  return <Table<T> columns={columnsWithMenu} {...rest} />;
+  return (
+    <Table<T>
+      getRowClassName={() =>
```
Comment (krawczykkatarzyna) [inline src/components/v5/common/TableWithMeatballMenu/TableWithMeatballMenu.tsx:22]: Can't it just be a string?

File: src/components/v5/common/TableWithMeatballMenu/utils.tsx [hunk 3]
```diff
@@ -12,6 +12,11 @@ export const makeMenuColumn = <T,>(
     cell: ({ row }) => {
       const props = getMenuProps(row);
 
-      return props ? <MeatBallMenu {...props} /> : undefined;
+      return props ? (
+        <MeatBallMenu
+          className="absolute top-4 right-4 md:top-auto md:right-auto"
```
Comment (joanna-pagepro) [inline src/components/v5/common/TableWithMeatballMenu/utils.tsx:17]: Is absolute needed on desktop?

**Review by joanna-pagepro — CHANGES_REQUESTED** [review]

File: src/components/v5/common/TableWithMeatballMenu/utils.tsx (hunk 3 above)
Comment (CzarekDryl) [reply src/components/v5/common/TableWithMeatballMenu/utils.tsx:17]: There is `md:relative` added in MeatBallMenu component

Comment (joanna-pagepro) [reply src/components/v5/common/TableWithMeatballMenu/utils.tsx:17]: So maybe you should add a wrapper here that will have absolute-related classes. This way you will separate the component's logic from the place where it's used

**Review by joanna-pagepro — APPROVED** [review]

File: src/components/v5/common/Table/types.ts (hunk 1 above)
Comment (joanna-pagepro) [reply src/components/v5/common/Table/types.ts:7]: The type is correct here. `?` doesn't mean that the function can also return undefined

**Pull request merged**